lib/actions/http.ex

```diff
@@ -250,7 +250,12 @@ defmodule WorkflowEngine.Actions.Http do
     end)
   end
 
-  @spec unwrap_response({:error, Mint.TransportError.t()} | {:ok, Req.Response.t()}, any) ::
+  @spec unwrap_response(
+          {:error, Req.TransportError.t()}
```

Comment:
Does this type spec work when only req 0.3 is installed? I would assume this leads to problems